xrviz/dashboard.py

```diff
@@ -81,8 +98,25 @@ def create_plot(self, *args):
             if self.var in list(sel_data.coords):  # When a var(coord) is plotted wrt itself
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+            c_low_val = self.kwargs['cmap lower limit']
+            c_upp_val = self.kwargs['cmap upper limit']
+
+            # It is better to set initial values as 0.1,0.9 rather than 0,1 i.e. (min, max)
```

Comment:
This is, of course, a matter of opinion. Quantiles probably take considerably longer to calculate.

Reply:
This is what I mean here by color balanced plot. I have tried setting `cmap lower limit` and `cmap upper limit` to  `sel_data.min()` and `max` and the graphs that we get are color imbalanced as follows:

![Screenshot from 2019-07-02 07-34-06](https://user-images.githubusercontent.com/30382331/60477631-6d5b2880-9c9d-11e9-87f0-031883af5dd7.png)

![Screenshot from 2019-07-02 07-35-46](https://user-images.githubusercontent.com/30382331/60477632-6df3bf00-9c9d-11e9-9551-5927831faba3.png)



If min or max is an outlier, it will result in the entire graph having almost same color.

Reply:
True in this case, but dependent on the data. We *could* allow selections to auto-fill the colour values at min/max, 10/90, 25/75... but I feel we are running out of space.